nebula/src/nebula/protos/faq.proto

```
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package faqservice;
```

Buf (1.54.0)

Files with package "faqservice" must be within a directory "faqservice" relative to root but were in directory "nebula/src/nebula/protos".

(PACKAGE_DIRECTORY_MATCH)